Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md

```diff
@@ -61,3 +61,45 @@ The integration allows the use of basic authentication in the requests.
 To enable basic authentication, a user and password have to be supplied in the Credentials parameters in the integration configuration.
 
 The server will then authenticate the requests by the `Authorization` header, expecting basic authentication encrypted in base64 to match the given credentials.
+
+
+## STIX types for STIX indicator Domain Object
+
+Some tools require the indicators in STIX to be a STIX indicator type and not an SCO. If that is the case you can select which indicator types will be converted into SDOs using a STIX Pattern for the indicator value.
+
+For example, when `STIX types for STIX indicator Domain Object` is not selected for IPs XSOAR will output the IP indicators as an SCO:
```

Comment:
```suggestion
For example, when `STIX types for STIX indicator Domain Object` is not selected for IPs, Cortex XSOAR will output the IP indicators as an SCO:
```

Reply:
I've removed the word XSOAR completely as XSIAM is also affected.